src/box/memtx_space.c

```diff
@@ -48,6 +48,7 @@
 #include "memtx_tuple_compression.h"
 #include "schema.h"
 #include "small/region.h"
+#include "box.h"
```

Comment:
Just like it's the case with Vinyl, there's the only place where we use `recovery_state` in memtx and it looks like it could be easily replaced with `memtx_recovery_state`.